active/progress/ProgressRing.md

```diff
@@ -146,25 +45,47 @@ isn't the type's default (for example an int-typed property that doesn't default
-| Value | double | Gets or sets the current setting of the range control, which may be coerced. | 
-| ShowPaused | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate a Paused state to the user.|
-| ShowError | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate an Error state to the user. |
-| Maximum | double | Defaults to 100. Gets or sets the highest possible Value of the range element. |
-| Minimum | double | Defaults to 0. Gets or sets the Minimum possible Value of the range element. |
-| ValueChanged | event | Event that occurs when the range value changes. |
+ProgressRing can now be used in scenarios where progress is determinate or indeterminates. With this additional capability, the determinate ProgressRing does not represent a "hung" state where the user cannot interact with the app. Previously, guidance recommended that ProgressRing only be used when the user cannot continue to interact with the app, but this is no longer the only use case and ProgressRing can be used in scenarios where user interaction can continue while the ring is spinning.
+
+Below are properties being added to ProgressRing. The IsIndeterminate property of ProgressRing will default to True for backwards compatability.
+
+|Name | Type | Description | Default | 
+|:--|:-:|:-:|:-:|
+| IsIndeterminate | boolean | Gets or sets a value that indicates whether the progress ring reports generic progress with a repeating pattern (indeterminate progress) or reports progress based on the Value property (determinate progress) | True | 
+| IsActive | boolean | Gets or sets a value that indicates whether the progress ring is being displayed to the end user. If IsActive is false, the ProgressRing is not shown, but space is reserved for it in the UI layout. To not reserve space for it, set its Visibility property to Collapsed. | True |
+| Value | double | Gets or sets the current setting of the range control, which may be coerced. The value ranges from 0 to 100. | 0 | 
+| DeterminateSource | AnimatedVisualSource | Gets or sets the Lottie animation file used for the determinate state of the ring. | N/A | 
+| IndeterminateSource | AnimatedVisualSource | Gets or sets the Lottie animation file used for the indeterminate state of the ring. | N/A | 
+| ValueChanged | event | Event that occurs when the range value changes. | N/A | 
 
 # API Details
 <!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+```xaml
+Unsealed runtimeclass ProgressRing : Microsoft.UI.Xaml.Controls.Control.RangeBase
+{
+    ProgressRing();
+ 
+    Boolean IsActive;
+    Boolean IsIndeterminate; 
+    Double Value; 
```

Comment:
This can be removed. The updated idl is already checked in [here](https://github.com/microsoft/microsoft-ui-xaml/blob/2ec9b1cdc15c3aa3d459b926f7e4888234b30387/dev/ProgressRing/ProgressRing.idl#L17).